spinn_front_end_common/interface/interface_functions/locate_executable_start_type.py

```diff
@@ -21,27 +21,38 @@
 
 
 class LocateExecutableStartType(object):
-    def __call__(self, graph, placements, graph_mapper=None):
+
+    def __call__(
+            self, graph, placements, executable_finder, graph_mapper=None):
         if not graph.vertices:
-            return [ExecutableType.NO_APPLICATION]
+            return [ExecutableType.NO_APPLICATION], {}
 
         binary_start_types = dict()
+        binary_to_start_type = dict()
+
         progress = ProgressBar(
             graph.n_vertices, "Finding executable start types")
         for vertex in progress.over(graph.vertices):
             # try to locate binary type, but possible it doesn't have one
             placement_binary_start_type = None
+            binary_name = None
             if isinstance(vertex, AbstractHasAssociatedBinary):
-                placement_binary_start_type = vertex.get_binary_start_type()
+                placement_binary_start_type = (
+                    vertex.get_binary_start_type())
+                binary_name = vertex.get_binary_file_name()
             elif graph_mapper is not None:
-                associated_vertex = graph_mapper.get_application_vertex(vertex)
-                if isinstance(associated_vertex, AbstractHasAssociatedBinary):
+                associated_vertex = (
+                    graph_mapper.get_application_vertex(vertex))
+                if isinstance(
+                        associated_vertex, AbstractHasAssociatedBinary):
                     placement_binary_start_type = \
                         associated_vertex.get_binary_start_type()
+                    binary_name = associated_vertex.get_binary_file_name()
 
             # check for vertices with no associated binary, if so, ignore
             if placement_binary_start_type is not None:
-                # update core subset with location of the vertex on the machine
+                # update core subset with location of the vertex on the
+                # machine
```

Comment:
Why split when not changed?

Reply:
fixed. im going to assume somewhere something was set to 79 and so was being flagged incorrectly